### ISSUE TITLE
nr-binder not binding to uesimtun0 interface

### ISSUE THREAD

**Issue by JustinnT** [aligungr/UERANSIM#340]
Hi Ali/Everyone

Thank you for developing UERANSIM. 

I followed this [tutorial](https://www.free5gc.org/installations/stage-3-sim-install/) and set up free5GC + UERANSIM. 

I can manually utilize the interface by typing `ping -I uesimtun0 google.com` and `sudo curl --interface uesimtun0 google.com`

![image](https://user-images.githubusercontent.com/42761012/120604206-620f6980-c47f-11eb-9056-8f4e372cf9ba.png)

![image](https://user-images.githubusercontent.com/42761012/120605600-d39be780-c480-11eb-8fac-ed91364d909c.png)

But when I use the `./nr-binder` tool for example: `./nr-binder 60.60.0.1 curl google.com`, it uses another interface as seen below.  
![image](https://user-images.githubusercontent.com/42761012/120605041-304ad280-c480-11eb-8456-8b94cd13962b.png)

Is this problem due to an error in my setup? Thank you !

**Comment by aligungr** [comment]
Hi @JustinnT 

`nr-binder` may not work if you use UERANSIM and free5gc in the same machine.

If they are already in separate machines, we should further investigate the problem.

Please first check whether or not this is the case.

Regards

**Comment by JustinnT** [comment]
Hi aligungr,

Thanks for the quick reply. I have fixed the issue. 

In my previous setup, the UERANSIM vm was cloned from free5gc. I imported a fresh vm for UERANSIM and updated cmake to 3.20.3 (I realised in the previous setup it was 3.10.3, and [wiki](https://github.com/aligungr/UERANSIM/wiki/Installation) required 3.17+). After doing the above I ran the nr-binder located in ~/UERANSIM/build and the correct interface is being used. 

I also noticed that there are 2 nr-binders, one in ~/UERANSIM/build and the other in ~/UERANSIM/tools. The one in the /tools folder seems to use the wrong interface. Is there any difference in these 2 nr-binders?

**Comment by aligungr** [comment]
@JustinnT 

You are right, it may be confusing. `nr-binder` in tools folder must **not** be used. It's automatically copied to build folder and `nr-binder` in builder folder should be used instead.

**Comment by noormohammedli** [comment]
Hello,

I have the same issue, 
I followed the tutorial as well, 
My free5GC on VM1 and  UERANSIM on VM2. 
nr-binder not binding to uesimtun0 interface 

Could you please help me? 
Thanks

**Comment by crashooo** [comment]
> 
Hi, @noormohammedli ,
I had the same problem as you， UERANSIM(v3.2.6) and free5GC are deployed on 2 VMs.
`ping -I uesimtun0 google.com` uses interface uesimtun0 correctly, but 
`./nr-binder 10.60.0.1 ping google.com` uses ens33 interface.
`10.60.0.1` is the UE IP address.
`[2022-11-16 10:31:54.752] [app] [info] Connection setup for PDU session[1] is successful, TUN interface[uesimtun0, 10.60.0.1] is up.`
